src/babashka/fs.cljc

```diff
@@ -773,7 +773,8 @@
   (-> path split-ext last))
 
 (defn split-paths
-  "Splits a string joined by the OS-specific path-seperator into a vec of paths."
+  "Splits a path list given as a string joined by the OS-specific path-seperator into a vec of paths.
+  On UNIX systems, the separator is ':', on Microsoft Windows systems it is ';'."
```

Comment:
We have a var for that in this namespace, maybe just mentioned that?

Reply:
Mentioning the var means I'll have to look at the var, which "hides" the actual value behind a java static field, so I then have to look at the documentation for that (where I copied part of the docstring from). In my opinion the docstring should make immediately clear, that this is not aboutbb splitting a path into its components, but splitting a list of paths into individual paths (it's hinted at by the function name: split-path*s*, not split-path, but I still tripped). If the docstring just mentions the var, I'd most likely assumed the var is `/` on UNIX and `\` on Windows (without looking at the var).